akka-protobuf/src/main/java/akka/protobuf/ByteString.java

```diff
@@ -373,7 +373,7 @@ private static ByteString readChunk(InputStream in, final int chunkSize)
    * Concatenate the given {@code ByteString} to this one. Short concatenations,
    * of total size smaller than {@link ByteString#CONCATENATE_BY_COPY_SIZE}, are
    * produced by copying the underlying bytes (as per Rope.java, <a
-   * href="http://www.cs.ubc.ca/local/reading/proceedings/spe91-95/spe/vol25/issue12/spe986.pdf">
+   * href="https://www.cs.ubc.ca/local/reading/proceedings/spe91-95/spe/vol25/issue12/spe986.pdf">
```

Comment:
This file is no longer found - I guess that makes 'https' as good as 'http', but perhaps you could find a working link?

Reply:
Hi @raboof ,
I am not able to find the up-to-date link for this one. I will try this next time.
Thanks